domains/*.banana.is-cool.dev.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "Wildcard for my NGINX Proxy Manager",
+
+    "domain": "is-cool.dev",
+    "subdomain": "*.banana",
```

Comment:
@WilliamDavidHarrison Do we support wildcard subdomains?

Reply:
No

Reply:
What I do then, you said you support wildcards, no?

Reply:
> Wildcard domains (like *.example.is-not-a.dev) are supported too, but the reason for their registration should be very clear and described in detail.

Reply:
so, updates?

Reply:
@BANanaD3V My bad, I thought this PR was for another domain service which doesn't allow wildcards, we do support them and will be happy to merge your PR. :)